src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/FlyoutTests/UnoSamples_Tests.Flyout.cs

```diff
@@ -190,5 +190,52 @@ public void FlyoutTest_Simple_FlyoutsCanBeDismissed()
 				return (rect2.CenterX, (rect1.Bottom + rect2.Y) / 2);
 			}
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.iOS, Platform.Android)]
+		public void FlyoutTest_FullPlacement()
+		{
+			Run("Uno.UI.Samples.Content.UITests.Flyout.Flyout_Simple");
+
+			_app.WaitForElement("FullOverlayFlyoutButton");
+			_app.Tap("FullOverlayFlyoutButton");
+
+			var flyoutContent = _app.WaitForElement("FullOverlayFlyoutContent").First();
+			var flyoutRect = flyoutContent.Rect.ToRectangle();
+			var flyoutCenter = new Point(flyoutRect.Left + flyoutRect.Width / 2, flyoutRect.Top + flyoutRect.Height / 2);
+
+			var visibleBounds = GetCorrectedVisibleBounds();
+			var boundsCenter = new Point(visibleBounds.Left + visibleBounds.Width / 2, visibleBounds.Top + visibleBounds.Height / 2);
+
+			if (flyoutCenter != boundsCenter)
+			{
+				Assert.Fail(new StringBuilder("Full placement flyout fails to centered in VisibleBounds:")
+					.AppendLine(PrettyPrint("flyoutContent", flyoutRect, flyoutCenter))
+					.AppendLine(PrettyPrint("visibleBounds", visibleBounds, boundsCenter))
+					.ToString()
+				);
+
+				string PrettyPrint(string label, Rectangle rect, Point center) =>
+					$"{label}: [LTWH: {rect.Left},{rect.Top},{rect.Width}x{rect.Height}][CenterXY: {center.X}, {center.Y}]";
+			}
+
+			Rectangle GetCorrectedVisibleBounds()
+			{
+				var bounds = this.GetVisibleBounds();
+				if (AppInitializer.GetLocalPlatform() == Platform.Android && bounds.Location == Point.Empty)
+				{
+					// for the android sample app, the coordinate system currently places point (0, 0) right below the below status bar
+					// we attempt to correct this with FitWindowsLinearLayout's offset
+					var layoutRect = _app.Query(x => x.Class("FitWindowsLinearLayout")).First().Rect.ToRectangle();
+					if (layoutRect.Size == bounds.Size)
+					{
+						bounds = layoutRect; // same as: new Rectangle(layoutRect.Location, bounds.Size);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5405523)